torchvision/datasets/video_utils.py

```diff
@@ -152,13 +152,13 @@ def _compute_frame_pts(self) -> None:
         with tqdm(total=len(dl)) as pbar:
             for batch in dl:
                 pbar.update(1)
-                clips, fps = list(zip(*batch))
+                batch_pts, batch_fps = list(zip(*batch))
```

Comment:
This is just renaming of local variables. The main thing is: what used to be called `clips` really really should not have been called `clips`. These are pts (presentation time stamps) for each frame in the videos. Nothing to do with a `clip` in this `VideoClips` context.